indico/htdocs/js/indico/modules/events/management.js

```diff
@@ -173,6 +176,44 @@
                 refreshPersonFilters();
             });
         }
+
+        $('.icon-ticket.js-show-regforms').qtip({
```

Comment:
`.js-show-regforms` should be enough; selecting `.icon-*` in jquery is not a good idea (that's a purely display-related class that should not have any meaning beyond that)